examples/surrounding_nitrogen_bonds/results/failed_molecules.txt

```diff
@@ -0,0 +1,35 @@
+Cc1cc(c(c(c1)C)C(CC(=NNC(=O)N)c2cccs2)C(C#N)C#N)C 10047 10047
```

Comment:
It seems like you are generating more results files. In general, unless these files are necessary to run the code, I think it is best practice to upload them to Google Drive, Dropbox, or similar, as those places are more permanent. It's not really a big deal to me though, unless there are going to be a lot of DANCE users who go through these tutorials.

Reply:
Ok sounds good! I'll make a Google Drive for them I think it will make the example look less cluttered